crypto/x509v3/v3_ia5.c

```diff
@@ -90,7 +90,7 @@ char *i2s_ASN1_IA5STRING(X509V3_EXT_METHOD *method, ASN1_IA5STRING *ia5)
 }
 
 ASN1_IA5STRING *s2i_ASN1_IA5STRING(X509V3_EXT_METHOD *method,
-                                   X509V3_CTX *ctx, char *str)
+                                   X509V3_CTX *ctx, const char *str)
```

Comment:
The cast of `str` inside this function looks a bit odd and casts away the `const`...although its ok really.